arcsi/api/item.py

```diff
@@ -402,3 +404,15 @@ def edit_item(id):
                 jsonify(item_details_partial_schema.dump(item)), 200, headers
             )
         return "Some error happened, check server logs for details. Note that your media may have been uploaded (to DO and/or Azurcast)."
+
+
+@arcsi.route("/<string:show_slug>/<episode_number>", methods=["GET"])
+def view_episode_archive(show_slug, episode_number):
+    do = DoArchive()
+    # TODO instead of json filtering,
+    # write actual query
+    # joining shows and items
+    # so we can limit date etc.
+    item_query = Show.query.filter_by(archive_lahmastore_base_url=show_slug).join(Item, Show.items).filter_by(number=episode_number).first().items
```

Comment:
Do we need `.items` objs here for the query execution below? Nice join!

Reply:
Yes, without the `.first()` it returns with a `BaseQuery`, and without the `.items` it returns a `Show` object

Reply:
This is a complex statement so of course there are multiple approaches but what about neither? If you just have the query that you then execute separately don't we save a transaction?